### DISCUSSION TITLE
華碩開發AI超音波人工智慧平台

### DISCUSSION THREAD

**Discussion by MingWei174** [1101-GS4719-C/Python-Community-GS4719-C#2055] [General]
https://tw.news.yahoo.com/%E8%8F%AF%E7%A2%A9%E5%B8%83%E5%B1%80-ai-%E9%86%AB%E7%99%82%E9%A0%98%E5%9F%9F-%E7%99%BC%E8%A1%A8%E5%85%A8%E6%96%B0%E8%B6%85%E9%9F%B3%E6%B3%A2%E4%BA%BA%E5%B7%A5%E6%99%BA%E6%85%A7%E5%B9%B3%E5%8F%B0-031418150.html
最近在新聞網上看到PC 品牌華碩近日發表全新超音波人工智慧平台，協助醫師、醫療機構與學術教育單位，普及雲端訓練及邊緣推論應用，藉 AI 大數據為醫療人員更快找出病灶。

我覺得此次推出的超音波人工智慧平台能讓醫師、學者在臨床研究中，大幅降低程式編碼門檻，省去人工判斷超音波影像的冗長作業流程，在醫療界算是相當大的突破。

我對AI應用在醫療領域抱有非常大的期待，不僅是因為它可以在醫療方面為我們帶來省時又事半功倍的效果，更能精確分析每位病患的身體情況，使整個國家醫療體系更加完善。華碩這次研發的AI超音波人工智慧平台的最大特色就是操作步驟簡單，在AI的幫助下，透過簡單易懂的圖像化介面，即可在不涉及程式碼的環境下，以最短時間和直觀操作，以最短時間和直觀操作，建立專屬於個人的超音波模型。

我覺得未來透過AI超音波人工智慧平台輔助，往後醫療機構在追蹤疾病或執行代謝症候群健康管理分析時，再也不必藉由人工逐筆判斷，就可以使醫療資源和人力資源運用的更有效率!

雖然現在有了AI超音波人工智慧平台輔助，但想這種智慧醫療類型的發展，往往會受限法規，醫療數據取得困難，另因醫療數據涉及病人個資，難以取得，使人工智慧在醫療領域的應用，比製造、交通等產業都慢。
且台灣健保補助有限，醫院缺乏推廣誘因人工智慧醫療服務必須取得主管機關、衛福部食藥署核准，才能用於醫療診斷。

我希望台灣政府也能因應AI時代的到來，修改會使AI發展受限的法律及各種醫療執行程序，使AI能更快速為台灣的醫療體系變得更加完善!

**Comment by hunteryang0108** [comment]
台灣的醫療一直是走在時代的尖端的，相信政府也能很快的修改法規程序讓這些技術得以實現的。過去這幾年有不少利用人工智慧輔助醫療行為的案例，對此我感到非常期待，期待未來有一天能攻克所有的醫療難題。

**Comment by Jerryguo888** [comment]
隨著華碩AI超音波平台的推出，其簡化了醫療影像分析過程，也提升診斷效率。這項技術的應用不僅省時高效，還能幫助醫療機構更精準地追蹤疾病。然而，現行法規和數據取得仍限制了AI在醫療領域的全面發展，希望政府能因應AI時代的來臨，調整相關法規，推動智慧醫療的普及。

**Comment by peresonacifer** [comment]
在未來，能把AI運用在醫療產業是一大進步，不只是因為AI能簡化操作上的困難，在病因調查和數據分析上也很有效率，然而，還是要考慮潛在的疑慮，如個資洩漏，所以病人相關的數據不能輕易地拿去給AI分析，這方面看能不能把個資的保護做得完善一點，讓法規能允許AI在醫療業上路。
